datadistillrSDK/project.py

```diff
@@ -0,0 +1,23 @@
+
+class project:
+    """
+    This is a class for getting project level data.
+
+    Attributes:
+        curr_session (session): Current session being used.
```

Comment:
Are the session details something that we'd want a user to see?  If not we should mark them as private by putting an underscore before the name and not documenting them.